src/site/stages/build/plugins/create-resources-and-support-section.js

```diff
@@ -323,6 +336,7 @@ function createResourcesAndSupportWebsiteSection(buildOptions) {
   }
 
   return files => {
+    excludeQaNodesThatAreNotStandalonePages(files);
```

Comment:
Here is where we basically unpublish QA nodes that have a `fieldStandalonePage` of false